### ISSUE TITLE
get global, local and unique variance

### ISSUE THREAD

**Issue by isglobal-brge** [tomlof/OnPLS#3]
I've been running your script and it seems that it runs without problems with my data. My question is: How can I get global, local and unique variance from the OnPLS model? (so far I'm including 3 tables but the idea is to use 7 - my main aim is to estimate the local variance between 1 table and the rest - I guess this can reduce the computing time, isn't it?)

**Comment by tomlof** [comment]
Yes, computing only the intersection between one block and the rest should speed up the computational time significantly. In fact, this should be just the globally joint model that you configure to predict the single block from the rest.

Currently, the global, local and unique variance can not be decomposed. I will add this functionality when I get the time, but from your description, it does not seem like you need the complete decomposition.

**Comment by isglobal-brge** [comment]
Thanks for the answer. Maybe I did not explain very well my question or I have not understood your answer. I was wondering how to get model information from your functions - An illustrative example would be very helpful

**Comment by tomlof** [comment]
Sorry for the late response. It's been very busy here. The estimators currently don't return or compute/store any statistics. This will be added when possible, but my time for this project is limited at the moment. (You're more than welcome to submit pull requests! ;-))

Is there any particular model information that you'd like to be able use?

**Comment by wenjun1992** [comment]
Dear Tommy,
Do you know which function in your Package can produce the below results?
1. The amount of variance explained in each component, for each data block, as well as cumulative variance explained by the model?
2. Correlation between model scores for each component, for each data block.

Thanks very much!

best wishes,
Cindy W

**Comment by Fred-White94** [comment]
Hi there,

I'd like to reopen this issue as I am trying to calculate and OnPLS model similarly to the work described in https://pubs.acs.org/doi/10.1021/acs.analchem.8b03205 in particular i would like to be able to estimate all local and unique components on top of the global ones. Which you guys appear to have code for!

Thank you in advance

**Comment by guokai8** [comment]
Hi @tomlof,
Does this one take care of? Can we get global, local, and unique variance now? And could you also explain the P, W, and T from the onpls? Can you also explain how to design the predComp ?
Thanks a lot!
K
